notebooks/openvino/question_answering_quantization.ipynb

```diff
@@ -0,0 +1,945 @@
+       "1                             BBC Radio 5 Live and 5 Live Sports Extra   \n",
+       "2                                                  5 Live Sports Extra   \n",
+       "3                                                         39 of the 43   \n",
+       "4                                                         39 of the 43   \n",
+       "5                                                                   44   \n",
+       "6                                                           T. J. Ward   \n",
+       "7                                                         41-yard line   \n",
+       "8                                                               Newton   \n",
+       "9                                                          three plays   \n",
+       "10                18 of 41 passes for 265 yards, with one interception   \n",
+       "\n",
+       "    INT8 F1  \n",
+       "0     22.22  \n",
+       "1     61.54  \n",
+       "2    100.00  \n",
+       "3     50.00  \n",
+       "4     50.00  \n",
+       "5    100.00  \n",
+       "6    100.00  \n",
+       "7     50.00  \n",
+       "8    100.00  \n",
+       "9     66.67  \n",
+       "10    18.18  "
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "results = []\n",
+    "metric = evaluate.load(\"squad_v2\" if VERSION_2_WITH_NEGATIVE else \"squad\")\n",
+    "\n",
+    "for item in validation_examples:\n",
+    "    id, title, context, question, answers = item.values()\n",
+    "    fp32_answer = hf_qa_pipeline(question, context)[\"answer\"]\n",
+    "    int8_answer = ov_qa_pipeline_ptq(question, context)[\"answer\"]\n",
+    "\n",
+    "    references = [{\"id\": id, \"answers\": answers}]\n",
+    "    fp32_predictions = [{\"id\": id, \"prediction_text\": fp32_answer}]\n",
+    "    int8_predictions = [{\"id\": id, \"prediction_text\": int8_answer}]\n",
+    "\n",
+    "    fp32_score = round(metric.compute(references=references, predictions=fp32_predictions)[\"f1\"], 2)\n",
+    "    int8_score = round(metric.compute(references=references, predictions=int8_predictions)[\"f1\"], 2)\n",
+    "\n",
+    "    if int8_score != fp32_score:\n",
+    "        results.append((question, answers[\"text\"], fp32_answer, fp32_score, int8_answer, int8_score))\n",
+    "\n",
+    "pd.set_option(\"display.max_colwidth\", None)\n",
+    "pd.DataFrame(\n",
+    "    results,\n",
+    "    columns=[\"Question\", \"Answer\", \"FP32 answer\", \"FP32 F1\", \"INT8 answer\", \"INT8 F1\"],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "58df445d-af43-4ba1-8195-7d8f00b8f82f",
+   "metadata": {},
+   "source": [
+    "### Model Size\n",
+    "\n",
+    "Quantization reduces the size of the model by up to four times. We save the FP32 PyTorch model and define a function to show the model size for the PyTorch and OpenVINO models."
```

Comment:
Do you know why the resulting model is only 2.39x smaller ? I would remove the "up to four times" sentence because of this

Reply:
I was also surprised to see it. It is because by default some values are added to ignored_scopes. I was unsure how to explain this, suggestions welcome, also @AlexKoff88